modules/shared/components/molecules/PostFooterCreation/PostFooterCreation.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import type { FC, ReactElement } from 'react';
+import type { IPostFooterCreation } from './IPostFooterCreation';
+import styles from './PostFooterCreation.module.css';
+import Privacy from '../Privacy/Privacy';
+import Divider from '../../atoms/Divider/Divider';
+
+/** 
+* @Notice you will use the handleChange functions to get and use the value of the toggler and the select
+* @ThingsNeededToBeDoneInParentComponent
+const [togglerIsChecked,setTogglerIsChecked] = useState(false);
+const handleTogglerChange = (e: React.ChangeEvent<HTMLInputElement>):void => { setTogglerIsChecked(e.target.checked) }<Test>;
+const handleSelectChange = (e: React.ChangeEvent<HTMLSelectElement>):void => {console.log(e.target.value)};
+*/
+const PostFooterCreation: FC<IPostFooterCreation.IProps> = ({
+  handleSelectChange,
+  togglerIsChecked,
+  handleTogglerChange,
+  disabled = true,
+  handleSubmitClick,
+  handleCancelClick,
+}): ReactElement => {
+  return (
+    <div className={styles.container}>
+      <Privacy
+        privacyOptions={['friends', 'family']}
+        withDivider
+        togglerIsChecked={togglerIsChecked}
```

Comment:
The functionality of the toggler is not working as expected

https://user-images.githubusercontent.com/8650435/121775603-0c2a7800-cb89-11eb-8fce-27ee8b2fc8ca.mov

Reply:
the toggler will work when the parent component passes the state and the on change function as a prop
![Capture](https://user-images.githubusercontent.com/68955535/121790723-728fb480-cbe2-11eb-9d96-2fcf7b063d1b.PNG)
i commented it in the code so whoever will use it will know what to do